database/modules.go

```diff
@@ -34,17 +35,10 @@ func (d *DB) BuildBlacklistedCollection() {
 	b.Run()
 }
 
-/*
- * Name:       BuildTBDCollection
- * Purpose:    Builds the TBD collection
- * Build Type:
- * Source:
- * comments:
- */
 func (d *DB) BuildTBDCollection() {
 	collection_name := d.r.System.TBDConfig.TBDTable
-	collection_keys := []string{"src", "dst"}
-	error_check := createCollection(d, collection_name, collection_keys)
+	collection_keys := []string{"uconn_id", "ts_score"}
```

Comment:
Index TBD by the score for fast sorting and the uconn_id for fast searching / joining